src/app/components/tile/index.html

```diff
@@ -175,4 +175,57 @@
     </sky-demo-page-code>
   </sky-demo-page-example>
 
+  <sky-demo-page-content sectionHeading="UX guidelines">
+    <p>
+      Tiles are containers to divide and group a large number of categories of varied content on a page. Use tiles when you have to provide access to a large number of content groupings where users would benefit from the capabilities to reorder, expand/collapse the tiles, or have personalization options for the content displayed.
+    </p>
+    <p>
+      See <a routerLink="/design/guidelines/content-containers">content container guidelines</a> for more details.
+    </p>
+  </sky-demo-page-content>
+
+  <stache-row>
+    <stache-column screenSmall="6">
+      <p>
+        Use tiles to divide a page with many categories of content, especially when the format or presentation of the content will differ across the categories.
+      </p>
+    </stache-column>
+    <stache-column screenSmall="6">
+      <div class="screenshot screenshot-success"><img src="~/assets/img/guidelines/tile/tile-example.png" style="width: 100%;" /></div>
+    </stache-column>
+  </stache-row>
+
+  <stache-row>
+    <stache-column screenSmall="6">
+      <p>
+        Don't use tiles to divide a page into a small number of sections.
```

Comment:
I swapped out "a small number of" for "unnecessary," but I don't know if that's clear enough even with the example image.

Reply:
I think I'd stick with "a small number." The wording is awkward, but it's more precise than "unnecessary," which could be interpreted to mean something else. ... Actually, now that I'm looking at this, I think maybe the problem isn't the wording but the guideline itself. I mean, what is "a small number"? Is it 3? 4? 5? I assume this is deliberately vague, but it would be more useful and clear if we set a number. Do you think it's worth reaching out to Todd to see if we can set a number? If he can set a number, this guideline could be "Don't use tiles to divide pages into fewer than XXX sections."

Reply:
It's hard to define an exact number, since the utility of the behaviors of Tiles rises with the size/complexity of each category, not just the number of categories. In other words, you probably need to exceed 4 or 5 categories (of simple details with a few fields each) before Tiles become appropriate, but you might only need 3 categories when they're deep/rich enough for different users to _repeatedly and reliably_ prioritize one over the others.

Thoughts on how to make that point?

Reply:
Maybe we can set 5 as the recommendation and then qualify it. What do you think of something along the lines of:
"Don't use tiles to divide pages into fewer than 5 sections. However, if tile content is complex enough for different users to repeatedly and reliably prioritize fewer tiles, you can divide pages into as few as 3 tiles."
Or:
"Don't use tiles to divide pages into fewer than 5 basic categories with simple details or fewer than 3 complex categories with content that is complex enough for different users to repeatedly and reliably prioritize the tiles."

Reply:
The second option above is a pretty complex sentence. I'll take Door #1.

Reply:
Done ✔️